pkg/ocmagenthandler/ocmagenthandler.go

```diff
@@ -3,13 +3,40 @@ package ocmagenthandler
 import (
 	"context"
 
+	ctrl "sigs.k8s.io/controller-runtime"
+
 	"github.com/go-logr/logr"
-	ocmagentv1alpha1 "github.com/openshift/ocm-agent-operator/pkg/apis/ocmagent/v1alpha1"
+	ocmagentv1alpha1 "github.com/openshift/ocm-agent-operator/api/v1alpha1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-//go:generate mockgen -source $GOFILE -destination ../../pkg/util/test/generated/mocks/$GOPACKAGE/interfaces.go -package $GOPACKAGE
+//go:generate mockgen -source $GOFILE -destination ../../pkg/util/test/generated/mocks/$GOPACKAGE/interfaces.go -package mocks
```

Comment:
Given this is generating a mock in a directory named after the original package, it goes against convention to name the package `mocks`. Any reason for the change?

Reply:
Agreed. I have a fix for this change. Will add it here.